proxy/hdrs/HdrToken.cc

```diff
@@ -70,7 +70,8 @@ static const char *_hdrtoken_strs[] = {
   "Newsgroups",   // NNTP
   "Organization", // NNTP
   "Path",         // NNTP
-  "Pragma", "Proxy-Authenticate", "Proxy-Authorization", "Proxy-Connection", "Public", "Range",
+  "Pragma", "Proxy-Authenticate", "Proxy-Authorization", "Proxy-Connection", "Public-Key-Pins-Report-Only", "Public-Key-Pins",
+  "Public", "Range",
```

Comment:
Can you please use clang no-format so that this array is consistently formatted with on entry per line?